sublime_jedi/tooltips/markdown.py

```diff
@@ -53,17 +53,20 @@ def _prepare_signature(self, docstring):
         """
         pattern = (
             '(?x)'
-            '^([\w\. \t]+\.[ \t]*)?'        # path
-            '(\w+)'                         # function / object
-            '[ \t]*(\([^\)]*\))'            # arguments
-            '(\s*->\s*.*$)?'                # annotation (rest of line)
+            '^([\w\. \t]+\.[ \t]*)?'    # path
+            '(\w+)'                     # function / object
+            '[ \t]*(\([^\)]*\))'        # arguments
+            '(?:'
+            '(\s*->\s*.*?)'             # annotation: -> Type
+            '(--|$)'                    # inline comment: -- blabla
+            ')?'
```

Comment:
oh, my.

I think we should guess docstring format properly here. Are you sure that docstring for `list().extend` is markdown formatted?